charts/forseti-security/values.yaml

```diff
@@ -48,7 +48,7 @@ server:
   image: "gcr.io/forseti-containers/forseti"
 
   # server.imageTag is the tag for the server container image.
-  imageTag: "v2.21.0"
+  imageTag: "v2.22.0"
```

Comment:
Should these be 2.23? OR will this be changed when the release is out?

Reply:
Same as below